contrib.json

```diff
@@ -345,6 +359,26 @@
         [ "git push upstream {{pr.base.ref}}",                                 "Push the changes to the main project" ],
         [ "git branch -D {{pr.user.login}}-{{pr.head.ref}}",                   "Delete the local branch used for merging" ]
       ]
+    },
+    "accept-as-is": {
+      "steps": [
+        { "prompt": "text", "id": "prNum",                             "desc": "What is the the pull request number?" },
+        { "get": "{{meta.urls.repo_api}}/pulls/{{prNum}}",             "desc": "Get the PR information", "id": "pr" },
+        { "get": "{{meta.urls.repo_api}}/pulls/{{prNum}}/commits",     "desc": "Get the PR commits to access author info", "id": "prCommits" },
+        [ "git checkout -b {{pr.user.login}}-{{pr.head.ref}} {{pr.base.ref}}", "Create a new branch for merging the changes" ],
+        [ "git pull --rebase {{pr.head.repo.ssh_url}}",                        "Merge and rebase the changes"],
+        [ "grunt test",                                                        "Run tests to make sure they still pass" ],
+        { "prompt": "text", "id": "line",                              "desc": "Describe this change in one line" },
+        [ "grunt chg-add:'{{line}} ([view](https\\://github.com/{{meta.org}}/{{meta.name}}/pull/{{prNum}}))'", "Add a line to the changelog" ],
+        [ "git add CHANGELOG.md",                                              "Add the changlelog change to be committed" ],
+        [ "git commit -a --author='{{prCommits.[0].commit.author.name}} <{{prCommits.[0].commit.author.email}}>' -m '{{line}}. closes #{{prNum}}'", "Commit the changes" ],
```

Comment:
Possibly `--reedit-message=<commit>`. But also how you have it now is perfectly fine.